### DISCUSSION TITLE
XYChart: BarGroup with Glyphs

### DISCUSSION THREAD

**Discussion by SimonPrato11** [airbnb/visx#1881] [Q&A]
Hi,

I'm using the visx/xychart package. I'm trying to create a grouped bar chart with glyphs as labels. 
While glyphs work fine for stacked bar charts, they don't work for grouped ones. 

The problem is that when the bars are grouped, the glyph can't access the position (x-axis) of each bar.
 I can't find a solution for this.

I would much appreciate any advice.

The original chart: 
<img width="1051" alt="Screenshot 2024-10-15 at 11 45 15" src="https://github.com/user-attachments/assets/22e213b1-3026-4804-b156-d5b052962dee">

A simpler codesandbox example:

https://codesandbox.io/p/sandbox/tooltip-visx-forked-9v7cp7?workspaceId=a5f704c6-2c6a-4a77-92a0-55b400b5f99c

<img width="550" alt="Screenshot 2024-10-15 at 13 35 05" src="https://github.com/user-attachments/assets/0deb37a1-38e8-43fd-976d-56befd28d241">

**Comment by bhanuu12** [comment]
Hi, 
I am facing the same issue, did you able to resolve it
